### PR TITLE
csv_importer: data cleanup, code reorganization, documentation

### PR DESCRIPTION
**Pull request by vbaruh**
Unfortunately this change became huge, it includes:
* Added/updated docs in the code and README files
* Data clean up
  * Now the subjects which are met with different abbreviations in the different
    csv files are imported with unified abbreviation for the subject. There's Subject 
    table containing the name and all possible abbreviations of all subjects.
  * The foreign language levels now are part of the subject id and subject name,
     previously there was a column `subject_specifier` where these levels were stored.
     This column no longer exist
  * Some DZI files contained columns with aggregations for all scores in 
     the CSV files (2ДЗИ and Общо). These columns were imported and were adding
     significant noise, they are no longer added
* DAGs related changes
  * now there dedicated DAGs for importing NVO and DZI data
  * there's bash script for installing the DAG definition on the airflow machine
  * helper script to generate airflow cli commands for importing data 
* The code was split in several py files in order to make it easier to read

### CODE REVIEW

File: csv_importer/csv_importer/models.py [hunk 1]
```diff
@@ -1,17 +1,33 @@
+"""
```
Comment (vbaruh) [inline csv_importer/csv_importer/models.py:1]: Docs about database tables

File: csv_importer/csv_importer/db_manage.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+from .runtime import is_dry_run, getLogger
+
+
+logger = getLogger(__name__)
+
+
+@dataclass
+class SubjectItem:
+    """
+    This class facilitates working with Subjects and the fact some of the
```
Comment (vbaruh) [inline csv_importer/csv_importer/db_manage.py:22]: docs about Subject table

File: csv_importer/csv_importer/db_manage.py [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+                )
+            )
+
+        return result
+
+
+    def _lang_variations(this: SubjectItem) -> list[SubjectItem]:
+        return _variations(this, language_levels)
+
+    default_subject_items = [
```
Comment (OggiH) [inline csv_importer/csv_importer/db_manage.py:121]: Just to check my understanding, if a new abbreviation appears in the raw data in the future, should we come in an manually add it to the respective `abbreviations` list in `default_subject_items`?

File: csv_importer/csv_importer/models.py [hunk 4]
```diff
@@ -36,6 +62,16 @@
     Column('place_id', ForeignKey('place.id'), nullable=False)
 )
 
+# Examination table representation one examination session or "Изпитна сесия".
```
Comment (OggiH) [inline csv_importer/csv_importer/models.py:65]: For DZIs, there are two sessions per year - May/June and August/September. The government portal currently includes only the May/June but the other data is available on request. So this is something we may want to include eventually noting that the data import will be different. So, I am wondering whether this calls for the inclusion of another column  e.g. `month` in the `examination` table. Probably worth discussing this in more detail offline.

**Review by OggiH — APPROVED** [review]

File: csv_importer/csv_importer/db_manage.py (hunk 3 above)
Comment (vbaruh) [reply csv_importer/csv_importer/db_manage.py:121]: That's correct. 
Also in order the new item to appear in the DB one should execute the following command:
`DB_URL="postgresql://....." ./csv_importer/app.py init-db`
Probably this part can be improved

File: csv_importer/csv_importer/models.py (hunk 4 above)
Comment (vbaruh) [reply csv_importer/csv_importer/models.py:65]: I agree we should discuss this.
I like the idea for `month` column.

I guess this additional autumn session will have fewer number of students attending, is that true? Are those students such who skipped the summer session or such that want to get better score?

**Pull request merged**